File: calendar.py
import re 
def calendar():
    day = input('What day? ')
    while (day < 30):
      activity = raw_input('What are you doing on that day? ')
      time = raw_input('What time is that activity? ')
      other = raw_input('What else are you doing? ')
      if (other = 'none'):
        print day
        return time + " " + activity
      else:
        time2 = raw_input('What time is that activity? ')
        print day
        return time + " " + activity + " " + time2 + " " + other

print calendar()

File: calendar.js
function calendar(){
    var day = prompt("What day? ", "day");
    while (day < 30){
        let activity = prompt('What are you doing on that day? ', "event");
        let time = prompt('What time is that activity?', "time");
        return time + " " + activity;
    }
}
/*function myFunction() {
    var person = prompt("Please enter your name", "Harry Potter");
    if (person != null) {
        document.getElementById("demo").innerHTML =
        "Hello " + person + "! How are you today?";
    }
}*/
console.log(calendar());